pkg/console/subresource/configmap/configmap.go

```diff
@@ -178,22 +177,6 @@ func getPluginsProxyServices(availablePlugins []*v1.ConsolePlugin) []consoleserv
 	return proxyServices
 }
 
-func GetTelemetryConfiguration(operatorConfig *operatorv1.Console, telemeterClientIsAvailable bool) map[string]string {
-	telemetry := make(map[string]string)
-	if len(operatorConfig.Annotations) > 0 {
-		for k, v := range operatorConfig.Annotations {
-			if strings.HasPrefix(k, telemetryAnnotationPrefix) && len(k) > len(telemetryAnnotationPrefix) {
-				telemetry[k[len(telemetryAnnotationPrefix):]] = v
-			}
-		}
-	}
```

Comment:
I think we lost all of these annotation properties?

Reply:
resolved